web/modules/custom/nys_registration/src/Controller/NysUserController.php

```diff
@@ -53,27 +78,31 @@ public function resetPassLogin($uid, $timestamp, $hash, Request $request) {
 
     user_login_finalize($user);
     $this->logger->notice(
-          'User %name used one-time login link at time %timestamp.',
-          [
-            '%name' => $user->getDisplayName(),
-            '%timestamp' => $timestamp,
-          ]
-      );
+      'User %name used one-time login link at time %timestamp.',
+      [
+        '%name' => $user->getDisplayName(),
+        '%timestamp' => $timestamp,
+      ]
+    );
     $this->messenger()->addStatus($this->t('You have just used your one-time login link. It is no longer necessary to use this link to log in. Please set your password.'));
     // Let the user's password be changed without the current password
     // check.
     $token = Crypt::randomBytesBase64(55);
     $request->getSession()->set('pass_reset_' . $user->id(), $token);
     // Clear any flood events for this user.
     $this->flood->clear('user.password_request_user', $uid);
+
+    // Bypass email TFA during password reset workflow.
+    $this->tempStoreFactory->get('email_tfa')->set('email_tfa_user_verify', 1);
```

Comment:
How does the `tempstore.private` service interact with config/state?

Reply:
@routinet If you search for `email_tfa_user_verify` in the `email_tfa` module, you can see how it is being used. I think typically `tempstore.private` is just modifying something for the current session. 

In this case, I'm assuming setting it to `1` just bypasses TFA for the reset form for the current user session.